.golangci.yaml

```diff
@@ -148,19 +134,14 @@ linters:
     - gosimple
     - govet
     - ineffassign
-    - interfacer
-    # - lll
```

Comment:
Can be enabled, see at https://github.com/networkservicemesh/api/pull/177

Reply:
Done